mavlink/Cargo.toml

```diff
@@ -31,6 +31,7 @@ mavlink-core = { version="=0.15.0", path = "../mavlink-core", default-features =
 num-traits = { workspace = true, default-features = false }
 num-derive = { workspace = true }
 bitflags = { workspace = true }
+bitflags-serde-legacy = { version = "0.1.1", optional = true}
```

Comment:
As far as I can see, this crate is just a temporary hack and I am unsure that it will be well-maintained in the future (especially since it's very unknown crate). Instead, we could simply downgrade the `bitflags` version which might be a cleaner solution, but I am opposed to that as well. If we change the major version, it should be totally acceptable to introduce breaking changes in the serialization/deserialization layouts. 

If `bitflags` has changed their ser/deser layout for good, the sooner we adapt to it the better.